taskcluster/scripts/build/macos_build.sh

```diff
@@ -24,6 +37,7 @@ print Y "Installing rust..."
 curl https://sh.rustup.rs -sSf | sh -s -- -y || die
 export PATH="$HOME/.cargo/bin:$PATH"
 
+print Y "Installing homebrew, cmake, ninja..."
```

Comment:
Ninja?

Reply:
https://ninja-build.org/ 
On taskcluster i'm using that as a generator  (`-GNinja`) instead of the default one - since it runs on all host's and we can use it for all targets, as it helped me get around msbuild and xcodebuild CI wierndess things :D